src/domains/aca-common/00_alerts.tf

```diff
@@ -0,0 +1,71 @@
+data "azurerm_api_management" "apim" {
+  name                = "${local.product}-apim"
+  resource_group_name = "${local.product}-api-rg"
+}
+
+resource "azurerm_resource_group" "rg_aca_alerts" {
+  count    = var.env_short == "p" ? 1 : 0
+  name     = "${local.project}-alerts-rg"
+  location = var.location
+  tags     = var.tags
+}
+
+data "azurerm_key_vault_secret" "monitor_aca_opsgenie_webhook_key" {
+  count        = var.env_short == "p" ? 1 : 0
+  name         = "aca-opsgenie-webhook-token"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_monitor_action_group" "aca_opsgenie" {
```

Comment:
why create a new one action grp ?
[should be better fetch it](https://github.com/pagopa/pagopa-infra/blob/c9612bede0ba65099480c682c15a96c002a30732/src/domains/fdr-app/00_monitor.tf#L25-L29)
and use it example like [here](https://github.com/pagopa/pagopa-infra/blob/c9612bede0ba65099480c682c15a96c002a30732/src/domains/fdr-app/00_alert.tf#L6)

Reply:
Ok to fetch it from an existing one. 
I was creating a new one here to have a per-domain action group in order to allow configuration for this specific domain (such as change api key/webhook url etc).
changing it

Reply:
fixed with https://github.com/pagopa/pagopa-infra/pull/2874/commits/94be51f6dad233212de569a00204c4627e58e920